packages/synapse-react-client/src/lib/containers/widgets/facet-nav/FacetNavPanel.tsx

```diff
@@ -531,7 +530,7 @@ const FacetNavPanel: React.FunctionComponent<FacetNavPanelProps> = (
             className={`FacetNavPanel__body${isModalView ? '--expanded' : ''}`}
             role="graphics-object"
           >
-            <SizeMe monitorHeight>
+            <SizeMe monitorHeight noPlaceholder>
```

Comment:
I was spot-checking our Plotly components and noticed that some don't appear on the first render in dev, but this isn't a new problem. `noPlaceholder` fixes it.